packages/waku/src/lib/middleware/dev-server.ts

```diff
@@ -139,6 +139,11 @@ const createMainViteServer = (
   };
 
   const loadServerModuleMain = async (id: string) => {
+    if (id === 'waku' || id.startsWith('waku/')) {
+      // HACK I don't know why this is necessary.
+      // `external: ['waku']` doesn't somehow work?
+      return import(id);
```

Comment:
Tbh, it does not feel right! I'd rather figure out how we can move default handler to use ssrLoadModule, that'd be way more robust, I guess.

But this can work too.

Reply:
> move default handler to use ssrLoadModule

Do you have time to try it?